src/services/news/events/index.js

```diff
@@ -0,0 +1,23 @@
+const logger = require('winston');
+
+const removeTeamNews = (app, newsService) => async (deletedTeam) => {
+	try {
+		await newsService.remove(null, {
+			query: {
+				target: deletedTeam._id,
+				targetModel: 'teams',
+			},
+		});
+	} catch (e) {
+		logger.warn(`Cannot remove news for team ${deletedTeam._id}`, e);
```

Comment:
this is an `error`?

Reply:
Yes, but it's not critical. We need some kind of policy regarding error logging.

Reply:
cc: @Metauriel @CeEv @tofixx